SMLHelper/Assets/CustomFabricator.cs

```diff
@@ -267,7 +269,46 @@ public void AddTabNode(string tabId, string displayText, Atlas.Sprite tabSprite,
                 CraftTreeLinkingNodes[tabId] = tab;
             });
         }
+#elif BELOWZERO
+
+        /// <summary>
+        /// Override this method if you want full control over how your custom craft tree is built up.<para/>
+        /// To use this method's default behavior, you must use the following methods to build up your crafting tree.<para/>
+        /// - <see cref="AddCraftNode(TechType, string)"/><para/>
+        /// - <see cref="AddCraftNode(string, string)"/><para/>
+        /// - <see cref="AddCraftNode(Craftable, string)"/><para/>
+        /// - <see cref="AddTabNode(string, string, Sprite, string)"/>
+        /// </summary>
+        /// <param name="craftTreeType"></param>
+        internal virtual void CreateCustomCraftTree(out CraftTree.Type craftTreeType)
```

Comment:
`CreateCustomCraftTree` method from both blocks can be combined

Reply:
https://github.com/SubnauticaModding/SMLHelper/pull/126/commits/749d9b35e5d1552863ead2120f124b42d0e0b7bb